imports/models/BoardFavoritesCollection.ts

```diff
@@ -0,0 +1,50 @@
+import { BaseCollection } from './BaseCollection'
+import { BoardRecord } from './BoardCollection'
+import { Meteor } from 'meteor/meteor'
+
+export interface BoardFavoriteRecord {
```

Comment:
interface naming convention here, always start with a `I` to indicate it's a interface. use `IBoardFavoriteRecord` here.